code/datums/view.dm

```diff
@@ -24,9 +25,12 @@
 
 /datum/view_data/proc/assertFormat()//T-Pose
 	winset(chief, "mapwindow.map", "zoom=0")
+	zoom = 0
 
 /datum/view_data/proc/resetFormat()//Cuck
-	winset(chief, "mapwindow.map", "zoom=[chief.prefs.read_preference(/datum/preference/numeric/pixel_size)]")
+	zoom = chief.prefs.read_preference(/datum/preference/numeric/pixel_size)
```

Comment:
can `chief` be null, as it's a client? 
or does view data ensure it's always there

Reply:
now that I think about it, I think the chief var here hangs references.
Will fix it to properly cleanup. Rule should be if the client doesn't exist, you'll have no way to call any of these procs
(That said something is really cursed in how byond handles clients as I'm sure you know. Maybe I should just use ?.)

Reply:
I don't know, any instance of `client.prefs` scares me for that reason. If you think it's fine though then it's w/e

Reply:
nah it shouldn't be like that you're right